docs/station_statistics.ipynb

```diff
@@ -0,0 +1,1397 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "037d0d40-b1d5-4724-92b2-a86c223b674e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "# Statistics\n",
+    "\n",
+    "This page presents statistics of solar irradiance monitoring stations using data from SolarStations.org."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3afafeb7-9ec2-4bd0-827d-edb4d3bd81c3",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.ticker as ticker\n",
+    "import numpy as np\n",
+    "from datetime import datetime as dt\n",
+    "from IPython.display import Markdown as md\n",
+    "from matplotlib.lines import Line2D"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "90ca945a-705d-4542-801a-3bd9c0bae7eb",
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "path = 'https://solarstations.org/_downloads/f85f41dda48ab008055bbfad91e578b2/SolarStationsOrg-station-catalog.csv'\n",
```

Comment:
This url seems strange.

I'm worried that the long alpha-numeric code will change upon a new release...

Reply:
I would not trust that slug to be reliable either.

It seems that the sphinx build processes this notebook _after_ the notebook that saves the "augmented" CSV file, so I suppose the path could be set to `./SolarStationsOrg-station-catalog.csv` to use the one that was generated earlier in the build.  It may not work as expected when running the notebook locally though.